libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -5,10 +5,17 @@
 import { Injectable } from '@nestjs/common';
 import { Stripe } from 'stripe';
 import { StripeClient } from './stripe.client';
+import { AuthLogger, AuthRequest } from '@fxa/shared/types';
 
 @Injectable()
 export class StripeManager {
-  constructor(private client: StripeClient) {}
+  constructor(
+    private client: StripeClient,
+    private db: any,
+    private push: any,
+    private profile: any,
```

Comment:
Should these 3 properties be optional instead? Will make it easier to instantiate the class for use cases where `customerChanged` isn't necessary.